src/pages/apps.js

```diff
@@ -1,5 +1,6 @@
 import React from "react";
+import Listing from "../components/apps/listing/Listing";
 
 export default function Apps() {
-    return <div>Launch an app from here</div>;
+    return <Listing baseId="apps" />;
```

Comment:
Not necessary for this PR, but at least for Data, I'm eventually going to have a `DataView` component that has everything, so things don't get added to the `Listing` component.  Not sure if you want to do the same.  Apps might be simpler and may not need it.

Reply:
Once we add few more UI elements in apps, I think the code will us more :)